Github Actions is not working because pdfchecker URL is broken

error:
```
+ BASE_URL=https://cdn.datalogics.com/www/pdflibinfo/grzqordg/che-lin64-MfeaURpvtQXzGKjY
+ DOWNLOADED_ZIP_FILENAME=PDF-CHECKER-Lin64.zip
+ INSTALL_DIR_PATH=/home/runner/work/fpdf2/fpdf2/PDF_Checker
++ mktemp -d /tmp/pdfchecker.XXXXXX
+ export TMPDIR=/tmp/pdfchecker.xdUJ3l
+ TMPDIR=/tmp/pdfchecker.xdUJ3l
+ wget --quiet https://cdn.datalogics.com/www/pdflibinfo/grzqordg/che-lin64-MfeaURpvtQXzGKjY/PDF-CHECKER-Lin64.zip
Error: Process completed with exit code 8.
```


I am disabling pdfchecker from the workflow until I can find a new URL

--- a/.github/workflows/continuous-integration-workflow.yml
+++ b/.github/workflows/continuous-integration-workflow.yml
@@ -48,10 +48,10 @@ jobs:
           # Using qpdf
           find . -name '*.pdf' | xargs -n 1 qpdf --check --password=fpdf2
           export PYTHONPATH=$PWD
-          # Using Datalogics PDF Checker: (disabled while download link is broken)
-          # scripts/install-pdfchecker.sh
-          # scripts/pdfchecker.py --process-all-test-pdf-files
-          # scripts/pdfchecker.py --print-aggregated-report
+          # Using Datalogics PDF Checker:
+          scripts/install-pdfchecker.sh
+          scripts/pdfchecker.py --process-all-test-pdf-files
+          scripts/pdfchecker.py --print-aggregated-report
           # Using VeraPDF:
           scripts/install-verapdf.sh
           scripts/verapdf.py --process-all-test-pdf-files

--- a/scripts/install-pdfchecker.sh
+++ b/scripts/install-pdfchecker.sh
@@ -6,14 +6,12 @@
 
 set -o pipefail -o errexit -o nounset -o xtrace
 
-BASE_URL=https://cdn.datalogics.com/www/pdflibinfo/grzqordg/che-lin64-MfeaURpvtQXzGKjY
-DOWNLOADED_ZIP_FILENAME=PDF-CHECKER-Lin64.zip
+BASE_URL=https://cdn.datalogics.com/evals
+DOWNLOADED_FILENAME=PDF-Optimizer-Checker-Linux64.bsx
 INSTALL_DIR_PATH=${1:-$PWD/PDF_Checker}
 export TMPDIR=$(mktemp -d /tmp/pdfchecker.XXXXXX)
 
-wget --quiet $BASE_URL/$DOWNLOADED_ZIP_FILENAME
-unzip $DOWNLOADED_ZIP_FILENAME
-rm $DOWNLOADED_ZIP_FILENAME
+wget --quiet $BASE_URL/$DOWNLOADED_FILENAME
 
 bsx_extract() {
     local bsx_filepath=${1?'Missing arg'}

